generators/server/templates/src/main/java/package/web/rest/util/_PaginationUtil.java

```diff
@@ -18,6 +18,9 @@
 -%>
 package <%=packageName%>.web.rest.util;
 
+import java.io.UnsupportedEncodingException;
```

Comment:
@FireWolf2007 sorry. I missed this. The imports should be under `<%_ if (searchEngine === 'elasticsearch') { _%>` otherwise it will become unused imports when ES is not used. It will affect sonar rating